public/app/plugins/datasource/dashboard/runSharedRequest.ts

```diff
@@ -17,11 +17,11 @@ export function isSharedDashboardQuery(datasource: string | DataSourceRef | Data
     // default datasource
     return false;
   }
-  if (datasource === SHARED_DASHBODARD_QUERY || (datasource as any)?.uid === SHARED_DASHBODARD_QUERY) {
+  if (datasource === SHARED_DASHBOARD_QUERY || (datasource as any)?.uid === SHARED_DASHBOARD_QUERY) {
     return true;
   }
   const ds = datasource as DataSourceApi;
-  return ds.meta && ds.meta.name === SHARED_DASHBODARD_QUERY;
+  return ds.meta && ds.meta.name === SHARED_DASHBOARD_QUERY;
```

Comment:
FYI I'm going to do a little gentle refactor of this function to remove these yucky type casts that just aren't necessary